WalletWasabi.Fluent/ViewModels/CoinJoinProfiles/CoinJoinProfilesViewModel.cs

```diff
@@ -46,6 +46,9 @@ public static CoinJoinProfileViewModelBase IdentifySelectedProfile(KeyManager ke
 		var currentProfile = new ManualCoinJoinProfileViewModel(keyManager);
 		var result = DefaultProfiles.FirstOrDefault(x => x == currentProfile) ?? currentProfile;
 
+		// Edge case: Update the PrivateCJProfile anonscore target, otherwise the randomly selected value will be displayed all time.
+		result.AnonScoreTarget = currentProfile.AnonScoreTarget;
```

Comment:
How about simply this?

```c#
// Edge case: Update the PrivateCJProfile anonscore target, otherwise the randomly selected value will be displayed all time.
if (result is PrivateCoinJoinProfileViewModel)
{
	result = new PrivateCoinJoinProfileViewModel(keyManager.AnonScoreTarget);
}
```

Feels more telling and also more secure as the `AnonScoreTarget` of the profile still cannot be changed externally.

Reply:
Uff, yes. This is much better.

Done https://github.com/zkSNACKs/WalletWasabi/pull/9593/commits/0e91ba8cf447137819ae71ab67f32eb28a3ff9ee